styles.css

```diff
@@ -0,0 +1,126 @@
+html {
+  background-color: rgba(229, 229, 229, 0.2);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "poppins", sans-serif;
+  width: 375px;
+  height: 718px;
+  border-bottom-left-radius: 79px;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+.home-container {
+  padding: 0 24px;
+  background-image: url("./icons/background.jpg");
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-position-y: 25px;
+}
+
+.logo {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 58px 24px 0 24px;
+  background-color: rgba(255, 255, 255, 1);
+  margin: 0;
+}
```

Comment:
- [ ] There is too much padding on the top of navbar, I recommend deleting extra padding from  top. 
=> `.logo { padding: 0 24px; }`
- [ ] You may be confused with the design where they indicated time battery level etc, but it should not be part of your project.